doc/articles/features/windows-system.md

```diff
@@ -12,7 +12,11 @@ This API is supported on iOS, Android, WASM and macOS.
 
 On iOS, Android and macOS the `ms-settings:` special URI is supported. 
 
-In case of iOS, any such URI opens the main page of system settings (there is no settings deep-linking available on iOS).
+#### Platform-specifics
+
+On iOS, launching the special URI opens the main page of system settings because deep-linking to specific settings is not available.
```

Comment:
This might be possible now? https://www.macstories.net/ios/a-comprehensive-guide-to-all-120-settings-urls-supported-by-ios-and-ipados-13-1/